docs/installation/troubleshooting/bootstrap_installation_errors.rst

```diff
@@ -0,0 +1,46 @@
+Bootstrap Installation Errors
+=============================
+
+Bootstrap installation fails with no straightforward reason
+-----------------------------------------------------------
+
+If during a MetalK8s installation you encounter a failure and the console
+output does not provide enough information to identify the cause of
+the failure, re-run the installation with the verbose flag (``--verbose``).
```

Comment:
(note for me: open a ticket to add information for reading the verbose logs without re-running the bootstrap script)

Reply:
Opened #3189